README.md

```diff
@@ -24,3 +24,13 @@ var authS3O = require('s3o-middleware');
 router.use(authS3O);
 app.use('/admin', router);
 ```
+If specific paths require auth:
+```js
+var express = require('express');
+var app = express();
+var router = express.Router();
+var authS3O = require('s3o-middleware');
+
+app.get('/', authS3O, router);
```

Comment:
Using `.use()` has the side-effect of e.g. `curl -X DELETE http://site.com/thing` would get handled, whereas before it wouldn't. Or, perhaps more realistic, you might want to s3o PUT, DELETE, or POST, but not GET for a given url: `.use()` won't allow you to do this